samples/kubernetes/end2end/docs/02-mysql.md

```diff
@@ -0,0 +1,41 @@
+## Setting up MYSQL Server
```

Comment:
Please add one paragraph that describe what kind of data you will be storing in MySQL database.

Reply:
Lily add this paragraph
"This document describes how to setup MySQL database in the Kubernetes cluster. In this sample we are using the MySQL database to store application data.  The WLS domain created in a later step will store its data to the database, e.g. persistent JMS messages."